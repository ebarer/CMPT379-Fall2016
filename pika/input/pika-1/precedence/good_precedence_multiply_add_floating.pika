exec {

  print 3.6 * 7.1 + 5.1, _n_.
  print 2.5 * 8.0 + 6.8, _n_.
  print 6.9 * 6.3 + 2.5, _n_.
  print 2.2 * 7.6 + 9.1, _n_.
  print 1.1 * 3.7 + 3.9, _n_.
  print 8.8 * 2.6 + 3.5, _n_.
  print 7.4 * 9.2 + 8.0, _n_.
  print 9.4 * 9.9 + 4.2, _n_.
  print 4.4 * 3.9 + 9.5, _n_.
  print 3.7 * 5.2 + 3.9, _n_.
  print 9.2 * 2.1 + 7.8, _n_.
  print 3.7 * 8.9 + 5.3, _n_.

  print 4.6 + 8.8 * 8.8, _n_.
  print 5.3 + 3.9 * 1.0, _n_.
  print 6.2 + 3.9 * 9.1, _n_.
  print 8.3 + 2.7 * 6.4, _n_.
  print 5.5 + 8.2 * 2.2, _n_.
  print 1.4 + 2.6 * 7.5, _n_.
  print 4.0 + 4.1 * 7.0, _n_.
  print 1.5 + 6.8 * 1.7, _n_.
  print 6.0 + 1.1 * 4.4, _n_.
  print 2.2 + 3.6 * 6.5, _n_.
  print 7.8 + 3.0 * 3.3, _n_.
  print 2.6 + 8.3 * 3.7, _n_.

  print 5.8 * 2.5 + 2.1 * 8.9, _n_.
  print 5.0 * 3.4 + 7.4 * 3.8, _n_.
  print 1.5 * 9.9 + 3.6 * 8.9, _n_.
  print 2.7 * 1.4 + 2.0 * 6.3, _n_.
  print 3.9 * 5.0 + 3.4 * 5.0, _n_.
  print 5.2 * 6.4 + 1.7 * 1.4, _n_.
  print 7.2 * 7.7 + 1.4 * 8.6, _n_.
  print 3.3 * 2.4 + 8.1 * 5.1, _n_.
  print 3.7 * 5.4 + 4.5 * 3.2, _n_.
  print 3.0 * 3.6 + 1.1 * 6.8, _n_.
  print 1.0 * 2.4 + 3.0 * 5.7, _n_.
  print 9.0 * 6.3 + 8.3 * 4.3, _n_.

  print 8.2 + 1.0 * 9.4 + 4.4, _n_.
  print 7.8 + 9.9 * 3.9 + 4.4, _n_.
  print 6.2 + 6.5 * 7.1 + 7.4, _n_.
  print 4.2 + 7.0 * 2.2 + 4.7, _n_.
  print 7.4 + 8.6 * 1.0 + 1.3, _n_.
  print 9.6 + 6.6 * 3.1 + 5.4, _n_.
  print 3.2 + 2.0 * 9.3 + 7.5, _n_.
  print 6.9 + 4.6 * 1.2 + 9.9, _n_.
  print 3.7 + 1.6 * 6.4 + 5.3, _n_.
  print 3.8 + 4.6 * 6.8 + 8.8, _n_.
  print 2.8 + 8.2 * 8.3 + 8.4, _n_.
  print 1.0 + 3.1 * 1.2 + 1.9, _n_.

}

