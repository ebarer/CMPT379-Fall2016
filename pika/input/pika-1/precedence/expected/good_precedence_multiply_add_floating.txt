30.66
26.8
45.97
25.82
7.97
26.38
76.08
97.26
26.66
23.14
27.12
38.23
82.04
9.2
41.69
25.58
23.54
20.9
32.7
13.06
10.84
25.6
17.7
33.31
33.19
45.12
46.89
16.38
36.5
35.66
67.48
49.23
34.38
18.28
19.5
92.39
22
50.81
59.75
24.3
17.3
35.46
29.3
22.32
19.24
43.88
79.26
6.62
